packages/api/tests/db-data-helpers/create-test-data.js

```diff
@@ -0,0 +1,178 @@
+'use strict';
```

Comment:
Having this helper in /tests/ is, I think, causing problems. The test runner is trying to find tests in here and failing, which I think might be causing a failure on the unit test step in CI. I'm moving this to `/db/src/test-data.ts` but open to other opinions if that doesn't seem right.